src/eth_rpc/servers/debug_rpc.rs

```diff
@@ -60,7 +59,27 @@ impl<P: EthereumProvider + Send + Sync + 'static> DebugApiServer for DebugRpc<P>
     }
 
     /// Returns an array of EIP-2718 binary-encoded receipts.
-    async fn raw_receipts(&self, _block_id: BlockId) -> Result<Vec<Bytes>> {
-        Err(EthApiError::Unsupported("debug_rawReceipts").into())
+    async fn raw_receipts(&self, block_id: BlockId) -> Result<Vec<Bytes>> {
+        Ok(self
+            .eth_provider
+            .block_receipts(Some(block_id))
+            .await?
+            .unwrap_or_default()
+            .into_iter()
+            .map(|receipt| {
+                Receipt {
+                    tx_type: receipt.transaction_type.to::<u8>().try_into().unwrap(),
+                    success: receipt.status_code.unwrap_or_default().to::<u8>() == 1,
+                    cumulative_gas_used: receipt.cumulative_gas_used.to::<u64>(),
+                    logs: receipt
+                        .logs
+                        .into_iter()
+                        .map(|log| Log { address: log.address, topics: log.topics, data: log.data })
+                        .collect(),
+                }
+                .with_bloom()
```

Comment:
`with_bloom` is quite a heavy computation, we actually store the receipts in our db with the blooms, so you could just get the bloom from receipt.bloom and avoid this computation.

Reply:
Yes right, just modified that to integrate the bloom directly without going through this slow method.